docs/pages/04.02-callbacks.md

```diff
@@ -197,3 +198,11 @@ filterItemsHandler(data, list) {
 	return list?.item?.label?.toLowerCase().startsWith(list?.inputValue?.toLowerCase());
 }
 ```
+
+### codeHeaderHandler()
```

Comment:
Can we make this naming more generic?  I'm thinking in the future we might use this for rendering other types of buttons.

Reply:
How about `customHeaderHandler` ? Assuming other buttons will also be a part of table header or control header.